client/lib/application/src/application-form/application-form.jsx

```diff
@@ -246,25 +248,27 @@ class ApplicationForm extends React.Component {
                         </div>
                         <div className='form-group'>
                             <label htmlFor='scm_url'>SCM URL</label>
-                            <small>Where you manage your source code.</small>
+                            <small>Where you manage your source code.
+                            The application SHOULD have a link to the main source code repository (git repo).</small>
                             <input
-                                placeholder='https://github.com/zalando-stups/pierone.git'
+                                placeholder='https://github.com/zalando-stups/pierone'
                                 id='scm_url'
                                 value={app.scm_url}
-                                pattern='^https?://.*'
-                                title='Please provide an URL with http or https.'
+                                pattern='^https://.*'
```

Comment:
Can we have the full pattern? `^https://<host>/<org>/<repo>` ? Something like `https://[^/]+/[^/]+/[^/]+` is enough I guess?

Reply:
done